### PR TITLE
feat(chart): add chart changes for v4.1.0

### PR DESCRIPTION
**Pull request by niladrih**

### CODE REVIEW

File: deploy/helm/charts/values.yaml [hunk 1]
```diff
@@ -118,7 +120,10 @@ helperPod:
     repository: openebs/linux-utils
     pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
-    tag: 4.0.0
+    tag: 4.1.0
```
Comment (tiagolobocastro) [inline deploy/helm/charts/values.yaml:123]: Was there any change to linux-utils?

**Review by tiagolobocastro — APPROVED** [review]

**Review by Abhinandan-Purkait — APPROVED** [review]

File: deploy/helm/charts/values.yaml (hunk 1 above)
Comment (Abhinandan-Purkait) [reply deploy/helm/charts/values.yaml:123]: We created a new tag for the alpine-bash and alpine-sh which also created a new linux-utils but no change in code.

Comment (niladrih) [reply deploy/helm/charts/values.yaml:123]: We've upgraded the base image for the container to alpine:3.20.1.

Comment (Abhinandan-Purkait) [reply deploy/helm/charts/values.yaml:123]: Ahh right, the base image is changed. So it's good to have this.

**Pull request merged**